# SOARING CONTROLLER POSITION IN ENU SOARING FRAME

uint64 timestamp			# time since system start (microseconds)

float32[3] pos                            # POSITION VECTOR IN SOARING ENU FRAME
float32[3] vel                            # VELOCITY VECTOR IN SOARING ENU FRAME
float32[3] acc                            # ACCELERATION VECTOR IN SOARING ENU FRAME